src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+const buttons = [
+  'AC', '+/-', '%', '÷',
+  '7', '8', '9', 'x',
+  '4', '5', '6', '-',
+  '1', '2', '3', '+',
+  '0', '.', '=',
+];
+
+const BuildArray = () => {
+  const buttonsArray = [];
+  buttons.forEach((button) => buttonsArray.push(
+    <button className="calc-buttons" type="button">{button}</button>,
+  ));
+  return buttonsArray;
+};
```

Comment:
[OPTIONAL]

- Great implementation of the calculator buttons. I love the method used. 👍 However, what do you think about keeping this implementation in its own component, say `Buttons` component? I believe this would make the `Calculator` component look cleaner and also make the new component reuseable across board. 😀

Reply:
@DammyShittu that is a good idea, can I pass the new `Buttons` as a "string" like that? ` <button className="calc-buttons" type="button">Buttons</button>`